$MirOS$

	Work around http://bugs.python.org/issue2585
	whose diff we cannot take because our urllib is older

--- Lib/urllib2.py.orig	Thu Jul 12 08:05:25 2007
+++ Lib/urllib2.py	Sun Oct  5 00:36:05 2008
@@ -491,7 +491,11 @@ class HTTPErrorProcessor(BaseHandler):
     handler_order = 1000  # after all other processing
 
     def http_response(self, request, response):
-        code, msg, hdrs = response.code, response.msg, response.info()
+        try:
+            code, msg, hdrs = response.code, response.msg, response.info()
+        except AttributeError:
+            # old python code
+            return response
 
         if code not in (200, 206):
             response = self.parent.error(
